src/v2/views/MonsterCollectionOverlay/index.tsx

```diff
@@ -66,9 +71,21 @@ function MonsterCollectionOverlay({ isOpen, onClose }: OverlayProps) {
             });
         }}
         onClose={onClose}
-        tip={tip?.nodeStatus.tip.index}
+        tip={tip.nodeStatus.tip.index}
         isLoading={isLoading}
-      />
+      >
+        {collection.stateQuery.monsterCollectionState && (
+          <Migration
+            tip={tip.nodeStatus.tip.index}
+            collectionState={collection.stateQuery.monsterCollectionState}
+            collectionSheet={collection.stateQuery.monsterCollectionSheet}
+            onActionTxId={(txId) => {
+              setLoading(true);
+              fetchStatus({ variables: { txId } });
```

Comment:
We're using the same refetching code when we're changing the deposit amount since what it does is refech when the tx is successfully staged.

This allows us to use the same loading screen made for changing the amount.